### PR TITLE
docs(ja): update jp-docs (commands/concepts/developings)

### PR DESCRIPTION
**Pull request by isleys**
periodic update for docs(ja)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: site/content/docs/commands/env-init.ja.md [hunk 1]
```diff
@@ -29,27 +29,44 @@ Import Existing Resources Flags
...
 ## 実行例
-AWS プロファイルの "default" に、デフォルト設定を使用して test Environment を作成します。
-
+AWS プロファイルの "default" 利用し、デフォルト設定を使用して test Environment を作成します。
 ```bash
 $ copilot env init --name test --profile default --default-config
 ```
 
-AWS プロファイルの "prod-admin" を利用して既存の VPC に prod-iad Environment を作成します。
+AWS プロファイルの "prod-admin" を利用して既存の VPC に prod-iad Environment を作成し、 CloudWatch Container Insights　を有効化します。
```
Comment (hkford) [inline site/content/docs/commands/env-init.ja.md]: Does this command import an existing VPC? I guess the `env init` command would ask you either of creating a VPC with default configuration or creating a VPC with specified configuration or importing a VPC.

File: site/content/docs/commands/pipeline-deploy.ja.md [hunk 2]
```diff
@@ -4,16 +4,18 @@ $ copilot pipeline deploy [flags]
 ```
 
 ## コマンドの概要
-`copilot pipeline deploy` は、ワークスペース内に全ての Service がデプロイ対象となるように Pipeline を作成/更新します。あわせてこの Pipeline のデプロイターゲットが Pipeline 用 Manifest にて Application と紐付けられた Environment 群となるように作成/更新されます。
+`copilot pipeline deploy` は、ワークスペース内の全ての Service に対する Pipeline をデプロイします。Pipeline 用 Manifest にて Application と紐付けられた Environment 群を利用します。
```
Comment (hkford) [inline site/content/docs/commands/pipeline-deploy.ja.md:7]: Does the `copilot pipeline deploy` command deploy a pipeline only for Copilot Services or Copilot Services and Copilot Jobs? L18 says the command deploys a pipeline for the services and jobs in your workspace. If so I'd like to imply that the services in your workspace are both Copilot Services and Copilot Jobs like 'ワークスペース内の Service や Job に対する Pipeline をデプロイします' or something. Welcome to discuss here 🙏

File: site/content/docs/concepts/services.ja.md [hunk 3]
```diff
@@ -23,17 +23,23 @@ Service のタイプを選択すると、Copilot は Dockerfile 内で記述さ
 インターネットからアクセス可能な Service を作る際の選択肢には次の２つがあります。
 
 * "Request-Driven Web Service" - Service 実行環境として AWS App Runner サービスを作成します。
-* "Load Balanced Web Service" - Service 実行環境として Appplication Load Balancer (ALB)、セキュリティグループ、ECS サービス (Fargate) を作成します。
+* "Load Balanced Web Service" - Service 実行環境として Appplication Load Balancer (ALB)、Network Load Balancer、または、その両方を作成し、セキュリティグループ、ECS サービス (Fargate) を利用します。
```
Comment (hkford) [inline site/content/docs/concepts/services.ja.md]: The original sentence here is 'an Application Load Balancer, a Network Load Balancer or both', so how about 'Application Load Balancer、Network Load Balancer、またはその両方' or 'Application Load Balancer (ALB)、Network Load Balancer (NLB)、またはその両方'?

File: site/content/docs/concepts/services.ja.md [hunk 4]
```diff
@@ -23,17 +23,23 @@ Service のタイプを選択すると、Copilot は Dockerfile 内で記述さ
...
 
 #### Request-Driven Web Service
 
 AWS App Runner を利用する Service で、受け付けるトラフィックに応じてオートスケールし、トラフィックがない場合は設定された最低インスタンス数までスケールダウンします。リクエスト量の大きな変化や恒常的な少ないリクエスト量が見込まれる HTTP サービスにとってもよりコスト効率の高い選択肢です。
+ECS とは異なり、 App Runner サービスはデフォルトでは VPC とは接続されていません。 出力トラフィックを VPC 経由でルーティングするには、
```
Comment (hkford) [inline site/content/docs/concepts/services.ja.md]: I'm sorry to bother you. Would you translate the term 'egress traffic' as 'Egress トラフィック'? Cf. [this line](https://github.com/aws/copilot-cli/blob/mainline/site/content/docs/manifest/rd-web-service.ja.md?plain=1#L152)

File: site/content/docs/concepts/services.ja.md [hunk 5]
```diff
@@ -23,17 +23,23 @@ Service のタイプを選択すると、Copilot は Dockerfile 内で記述さ
...
+マニフェスト内の[`network`](../manifest/rd-web-service.ja.md#network)フィールドを設定します。
 
 #### Load Balanced Web Service
 
-Application Load Balancer をトラフィックの入り口として Fargate 上でタスクを実行する ECS サービスです。安定したリクエスト量が見込まれる場合、Service から VPC 内のリソースにアクセスする必要がある場合、あるいはより高度な設定の必要がある場合に適した選択肢です。
+Application Load Balancer、Network Load Balancer、または両方をトラフィックの入り口として Fargate 上でタスクを実行する ECS サービスです。
+安定したリクエスト量が見込まれる場合、Service から VPC 内のリソースにアクセスする必要がある場合、あるいはより高度な設定の必要がある場合に適した HTTP まはた TCP サービスの選択肢です。
```
Comment (hkford) [inline site/content/docs/concepts/services.ja.md]: 'まはた' to 'または'

File: site/content/docs/concepts/services.ja.md [hunk 6]
```diff
@@ -23,17 +23,23 @@ Service のタイプを選択すると、Copilot は Dockerfile 内で記述さ
...
 
-Service でインターネット側からのリクエストを捌きたいですか？__Load Balanced Web Service__ を選べば、Copilot は Application Load Balancer やセキュリティグループ、そしてあなたの Service を Fargate で実行するための ECS サービスを作成します。
+Application Load Balancer は Environment レベルのリソースであり、Environment 内の全ての Load Balanced Web Service で共有されることに注意しましょう。
+詳細については、[こちら](environments.ja.md#load-balancers-and-dns)を確認してください。対照的に、 Net Work Load Balancer は Service レベルのリソースであり、 Service 間では共有されません。
```
Comment (hkford) [inline site/content/docs/concepts/services.ja.md]: It seems there is an extra space between 'Net' and 'Work'.

File: site/content/docs/developing/domain.ja.md [hunk 7]
```diff
@@ -34,23 +43,23 @@ http:
...
 裏側では Copilot は
 
 * Application を作成したアカウント内で `${AppName}.${DomainName}` というサブドメイン用のホストゾーンを作成し
 * Environment があるアカウント内で `${EnvName}.${AppName}.${DomainName}` という新しい Environment 用のサブドメインのために別のホストゾーンを作成し
-* Environment 用のサブドメインに使う ACM 証明書の作成と検証を行い
-* HTTPS リスナーと証明書を関連づけて HTTP のトラフィックを HTTPS にリダイレクトし
+* Environment 用のサブドメインに使う ACM 証明書の作成と検証し
+* ACM 証明書を
+    - エイリアスが Application Load Balancer　(`http.alias`) として利用されている場合、HTTPS リスナー と関連づけて HTTP のトラフィックを HTTPS にリダイレクトし、
+    - エリアスが `nlb.alias` として利用されていて、 TLS ターミーネーションが有効な場合、 Network Load Balancer の TLS リスナー
```
Comment (hkford) [inline site/content/docs/developing/domain.ja.md]: 'Network Load Balancer の TLS リスナーと関連付け' or something?

File: site/content/docs/developing/publish-subscribe.ja.md [hunk 8]
```diff
@@ -1,17 +1,17 @@
 # パブリッシュ / サブスクライブ アーキテクチャ
 
-Copilot [Worker Services](../manifest/worker-service.ja.md)は、すべてのサービスタイプとジョブタイプに共通する `publish` フィールドを利用して、サービス間でメッセージを受け渡すためのパブリッシュ / サブスクライブロジックを簡単に作成することができます。
+Copilot [Worker Services](../manifest/worker-service.ja.md)は、すべてのサービスタイプとジョブタイプに共通する `publish` フィールドを利用して、サービス間でメッセージを受け渡すためのパブリッシュ / サブスクライブロジックを簡単に作成できます。
```
Comment (hkford) [inline site/content/docs/developing/publish-subscribe.ja.md]: Are the terms 'サービス' and 'ジョブ' supposed to be translated as the concept word of Copilot? Cf. [this page](https://github.com/aws/copilot-cli/blob/mainline/site/content/docs/concepts/services.ja.md?plain=1#L15)

**Review by hkford — COMMENTED** [review]
Thank you @isleys for the pull request! I left some comments.

File: site/content/docs/commands/env-init.ja.md (hunk 1 above)
Comment (isleys) [reply site/content/docs/commands/env-init.ja.md]: The translation of importing an existing VPC is wrong.

**Review by hkford — APPROVED** [review]
LGTM! Thank you @isleys for your contributions!

**Review by huanjani — APPROVED** [review]
Thanks so much!

**Pull request merged**